vector/src/main/java/im/vector/adapters/VectorMessagesAdapter.java

```diff
@@ -1800,6 +1773,10 @@ private boolean isSupportedRow(MessageRow row) {
 
         boolean isSupported = VectorMessagesAdapterHelper.isDisplayableEvent(mContext, row);
 
+        if (!isSupported) {
+            Log.w(LOG_TAG, "Unsupported row. Event type: " + event.getType());
+        }
+
         if (isSupported && TextUtils.equals(event.getType(), Event.EVENT_TYPE_STATE_ROOM_MEMBER)) {
```

Comment:
Can we use here `else if` and remove check on `isSupported`?

Reply:
Yes, but I prefer to keep the algo like this, and this is just for a log